### ISSUE TITLE
Raspberry Pi boot time

### ISSUE THREAD

**Issue by vmayoral** [rancher/os#1337]
Hello everyone, great things happening around RancherOS. Nicely done.

 I'm testing rancherOS version `0.6.1` in a `Raspberry Pi 3`. According to the [docs](http://docs.rancher.com/os/) the boot time is about 5-10 seconds however I obtained something like [this](https://gist.github.com/vmayoral/191a1c5fe473e74e5bbeab432acb5117) with a prompt close to second 40.

I guess the docs were meant for x86. Can anyone comment on this? Any way to lower the boot time for user containers?

Many thanks!

**Comment by joshwget** [comment]
Correct, the referenced boot time is meant for typical x86 servers. It'll be slower booting pretty much any operating system on a Raspberry Pi.

We're doing some general work to improve boot time such as moving some remaining code from Bash to Go and removing unnecessary system services. These changes should have a proportionally higher affect for devices like a Raspberry Pi.
